packages/custom_lint/bin/custom_lint.dart

```diff
@@ -16,6 +16,19 @@ Future<void> entrypoint([List<String> args = const []]) async {
       help: 'Treat warning level issues as fatal',
       defaultsTo: true,
     )
+    ..addOption(
```

Comment:
This flag seems useless at the moment to me.

Unless we have at least 2 formats, we shouldn't have it

Reply:
Sure, I can remove this and re-add it in the next PR with the JSON format.

Reply:
Done